gdb/stub/target/microwatt.py

```diff
@@ -255,6 +255,9 @@ def register_read(self, regnum: int) -> bytes:
         value = value[0 : reg.size // 8]
         return value
 
+    def register_write(self, regnum, data):
+        pass
```

Comment:
I'd change `pass` to `raise Exception("Should be implemented!")` here to avoid silent failures. This can be source of couple hair-pulling hours trying to figure out why we still see the old value even if we changed it (with no error).